cvmfs/swissknife_lease_json.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#ifndef CVMFS_SWISSKNIFE_LEASE_JSON_H_
+#define CVMFS_SWISSKNIFE_LEASE_JSON_H_
+
+#include "swissknife_lease_curl.h"
+
+#include <string>
+
+bool ParseAcquireReply(const CurlBuffer& buffer, std::string* session_token);
```

Comment:
Ok, makes sense.  I don't think we should add more namespaces, I have the feeling they are rather adding to the confusion...